_tools/ic_people_website.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: IC People Website
+category: tools
+order: 2
+number_sections: TRUE
+---
+
+EPFL IC provides personal research page hosting under the `ic-people.epfl.ch/~name` domain.
+
+- Protocol: sftp
+- Host: ic-people-files.epfl.ch
+- Username: Gaspar Username
+- Password: Gaspar Password
+- Index folder: public_html
+
```

Comment:
I agree, however having the bare-minimum information about the service is nice to have. I suggest to keep this info since it should not be changed (most probably).

Reply:
Fair enough.